### ISSUE TITLE
GET /generate-report - Returns report url

### ISSUE THREAD

**Issue by fescobar** [fescobar/allure-docker-service#84]
Returns `report_url` for having bidirectional traceability.
```
{
   "data":{
      "allure_results_files":[
         "any_file.json"
      ],
      "report_url":"http://localhost:5050/projects/default/reports/5/index.html"
   },
   "meta_data":{
      "message":"Report successfully generated for project_id 'default'"
   }
}
```

**Comment by fescobar** [comment]
@IgorDodukh I've released a beta version with this feature added `image: "frankescobar/allure-docker-service:beta"`. Remove a previous local beta if you have one.

To resolve the report url generated from this version the `latest` report is exactly the same as the last execution number.

Please, test it and let me know if there is a problem or not.

**Comment by IgorDodukh** [comment]
@fescobar I've tried `beta` version and it looks good for me. No problems were found.
I really appreciate your efforts and quick feedback. Thank you!

**Comment by fescobar** [comment]
@IgorDodukh I've re-released `2.13.3` including this feature with examples. Please, remove your previous `2.13.3` if you have one.
